protobuf/lib/src/protobuf/extension_registry.dart

```diff
@@ -92,6 +94,7 @@ T _reparseMessage<T extends GeneratedMessage>(
     T message, ExtensionRegistry extensionRegistry) {
   T result = message.createEmptyInstance();
```

Comment:
Can we avoid creating the empty instance and the shallow copy until we find something?